kostyor/tests/unit/rest_api/test.py

```diff
@@ -339,3 +340,61 @@ def test_discover_cluster_discovery_failed_error_response(self,
                                               auth_url='http://9.9.9.9')
         fake_session.assert_called_once_with(auth=password_mock)
         fake_discover.assert_called_once_with()
+
+    @mock.patch.object(db_api, 'get_cluster', mock.Mock())
+    @mock.patch.object(db_api, 'get_hosts_by_cluster', mock.Mock())
+    def test_host_list_cluster_exists_success(self):
+        fake_hosts = ['hostname_1', 'hostname_2']
```

Comment:
Ok, it will be better to change mock values. As for testing REST API and DB layer together, it seems to me that it would be integration of functional testing, not unit:)